clients/cli/cmd/internal/init.go

```diff
@@ -143,7 +143,11 @@ func (cmd *InitCommand) askForToken() error {
 }
 
 func (cmd *InitCommand) setToken(token string) {
-	cmd.YAML.AccessToken = token
+	saveTokenToConfig := ""
+	_ = prompt.WithDefault("Do you want to save the token to the config file? Otherwise you will have to provide it every time you run the command either through a command argument or through an environment variable (y/n)", &saveTokenToConfig, "n")
```

Comment:
@jablan could you mention the argument and environment variable? might make it a little easier when it already shows you how to pass it